src/scripts/main.js

```diff
@@ -4,6 +4,16 @@ let flkty = null;
 let carouselElem = null;
 const Flickity = require('flickity');
 
+preventSubmit();
```

Comment:
Why don`t you use [preventDefault](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)? Now page still reloading after form submiting.